docs/about/all/releases/security-release-notes-202503.0.md

```diff
@@ -240,12 +240,12 @@ A secret related to the "Remember Me" functionality was found to be hardcoded in
 
 2. Update `.git.docker` with hash commit `ac17ea980d151c6b4dd83b7093c0c05a9205c244` or higher.
 
-3. Add `SPRYKER_CUSTOMER_REMEMBER_ME_SECRET` environment variable in the cloud. For instructions, see [Add variables in the Parameter Store](/docs/ca/dev/add-variables-in-the-parameter-store.html).
+3. Add `CUSTOMER_REMEMBER_ME_SECRET` environment variable in the cloud. For instructions, see [Add variables in the Parameter Store](/docs/ca/dev/add-variables-in-the-parameter-store.html).
```

Comment:
pls check my comment in [Slack](https://spryker.slack.com/archives/C3KMWM1UZ/p1743579990895279?thread_ts=1743575195.374069&cid=C3KMWM1UZ)

Reply:
@matweew I don't think I have access to Slack anymore :)

Reply:
Sorry, I thought the author was our internal developer. So the question is does not Spryker Cloud generate it out of the box for every project?

Reply:
That is a very good question. It is not generated for our project (existing project), but I am not sure if it will be generated for new projects.

But, I think this documentation is intended for existing customer that are using older version of Spryker. So, it is safe to assume that this ENV variable is not generated for them.

You can confirm this with Spryker Cloud team maybe?